libs/filabridge/src/Variant.cpp

```diff
@@ -218,8 +214,16 @@ utils::Slice<const Variant> getUnlitVariants() noexcept {
     return { details::gUnlitVariants.data(), details::gUnlitVariants.size() };
 }
 
-utils::Slice<const Variant> getDepthVariants() noexcept {
-    return { details::gDepthVariants.data(), details::gDepthVariants.size() };
+utils::Slice<const Variant> getLitDepthVariants() noexcept {
+    return { details::gLitDepthVariants.data(), details::gLitDepthVariants.size() };
+}
+
+utils::Slice<const Variant> getUnlitDepthVariants() noexcept {
+    return { details::gUnlitDepthVariants.data(), details::gUnlitDepthVariants.size() };
+}
+
+utils::Slice<const Variant> getPostProcessVariants() noexcept {
+    return { details::gPostProcessVariants.data(), details::gPostProcessVariants.size() };
```

Comment:
Something is odd here. A depth variant can't be lit or unlit; it's just a depth variant. In fact the DIR, DYN and SRE bits don't have meaning if DEP is on.